tmp/models.py

```diff
@@ -1,20 +1,43 @@
-from sqlalchemy import Column, DateTime, String
+from sqlalchemy import Column, ForeignKey, String, Table
 from sqlalchemy.ext.declarative import declarative_base
-from sqlalchemy.sql import func
+from sqlalchemy.orm import relationship
 
 
 Base = declarative_base()
 
+association_table = Table(
+    'user_place_association', Base.metadata,
+    Column('users', String(40), ForeignKey('users.ip')),
+    Column('places', String(40), ForeignKey('places.code'))
+)
+
 
 class User(Base):
     __tablename__ = 'users'
 
     ip = Column(String(40), primary_key=True)
-    choice = Column(String(10), nullable=False)
-    timestamp = Column(
-        DateTime(timezone=True), nullable=False,
-        server_default=func.now())
+    places = relationship(
+        "Place",
+        secondary=association_table,
+        back_populates="users"
+    )
+
+
+class Place(Base):
+    __tablename__ = 'places'
 
-    def __repr__(self):
-        return (f"<User ip={self.ip} choice={self.choice}"
-                f" timestamp={self.timestamp}>")
+    code = Column(String(40), nullable=False, primary_key=True)
+    place = Column(String(20), nullable=False)
```

Comment:
I think `name` is better than `place` because it is Places' name.
place.name = '~~'